packages/ui-test-utils/src/utils/expect.ts

```diff
@@ -33,8 +33,6 @@ import assertions from './assertions'
 const init = (chai: Chai.ChaiStatic) => {
   // eslint-disable-next-line no-param-reassign
   chai.config.truncateThreshold = 999
-  // eslint-disable-next-line no-param-reassign
-  chai.config.includeStack = true
```

Comment:
this was just some totally useless stack trace, displaying lines in Karma and Mocha